### PR TITLE
Feat: actionable pending txs on the Dashboard

### PR DESCRIPTION
**Pull request by katspaugh**
## What it solves

Part of #2510

## How this PR fixes it

Shows only signable and executable queued txs, not just any queued txs. Also displays a ✔️ or 🚀 button if a tx can be executed or signed.

Screenshot 2023-09-18 at 17 25 00

### CODE REVIEW

File: src/components/dashboard/PendingTxs/PendingTxListItem.tsx [hunk 1]
```diff
@@ -19,6 +23,10 @@ type PendingTxType = {
 const PendingTx = ({ transaction }: PendingTxType): ReactElement => {
   const router = useRouter()
   const { id } = transaction
+  const { safe } = useSafeInfo()
+  const wallet = useWallet()
+  const signable = wallet ? isSignableBy(transaction, wallet.address) : false
+  const executable = wallet ? isExecutable(transaction, wallet?.address, safe) : false
```
Comment (usame-algan) [inline src/components/dashboard/PendingTxs/PendingTxListItem.tsx:29]: I would suggest to keep the naming convention of the other places we use these functions: `canSign`, `canExecute`

**Review by usame-algan — APPROVED** [review]
Nice one 💯

**Comment by francovenica** [comment]
Looks goood to me:

I tested in a safe 3/3:
The users can only see transactions that they can confirm. 
They can see tx that are 1 signature away from execution and the icon shown is the "rocket" to execute right away.
A fully signed tx shows up again with the "Rocket" icon to execute

**Comment by francovenica** [comment]
Reopening this one because of an issue found:

I created a tx with an owner and he can see the tx in the list. Is expected that he shouldn't see the tx since he already signed during creation

The safe:
https://pending--walletweb.review-wallet-web.5afe.dev/home?safe=gor:0xA601D7b958A7Cdd82B5Cf5230D6C40876E9e27C4

The owner: (see if impersonator works)
gor:0x7724b234c9099C205F03b458944942bcEBA13408

Gif:
tx list

**Comment by katspaugh** [comment]
When there are no signable/executable txs, it will display any queued txs, so it's expected.

**Pull request merged**